pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -811,6 +826,34 @@ protected boolean buildSegmentAndReplace() {
     return true;
   }
 
+  private void closeKafkaConsumers() {
+    closePartitionLevelConsumer();
+    closeStreamMetadataProvider();
+    if (_acquireConsumerSemaphore.compareAndSet(true, false)) {
+      _partitionConsumerSemaphore.release();
+    }
+  }
+
+  private boolean closePartitionLevelConsumer() {
```

Comment:
Why is this returning a boolean, when the callers are not considering the return value?

Reply:
Updated to void.